container_api/src/config.rs

```diff
@@ -30,26 +30,31 @@ use toml;
 #[derive(Deserialize)]
 pub struct Configuration {
     /// List of Agents, this mainly means identities and their keys
-    pub agents: Option<Vec<AgentConfiguration>>,
+    #[serde(default)]
+    pub agents: Vec<AgentConfiguration>,
     /// List of DNAs, for each a path to the DNA file
-    pub dnas: Option<Vec<DNAConfiguration>>,
+    #[serde(default)]
+    pub dnas: Vec<DNAConfiguration>,
     /// List of instances, includes references to an agent and a DNA
-    pub instances: Option<Vec<InstanceConfiguration>>,
+    #[serde(default)]
+    pub instances: Vec<InstanceConfiguration>,
     /// List of interfaces any UI can use to access zome functions
-    pub interfaces: Option<Vec<InterfaceConfiguration>>,
+    #[serde(default)]
+    pub interfaces: Vec<InterfaceConfiguration>,
     /// List of bridges between instances
-    pub bridges: Option<Vec<Bridge>>,
+    #[serde(default)]
+    pub bridges: Vec<Bridge>,
```

Comment:
All these defaults can't be right. Is an empty configuration a valid one?

Reply:
@sphinxc0re sure, it's OK to just not specify bridges in the config if you don't need them right?

It makes it so absence of a list corresponds to the empty list

Reply:
@sphinxc0re which would you recommend should be required? `agents` and `dna`?

Reply:
I'm not sure, that's why I'm asking. What exactly does the agent need to get running?

Reply:
@sphinxc0re you're right, according to `check_consistency`, agents, dnas, and instances should be required, so I changed it.

Reply:
👍